woof-code/rootfs-petbuilds/connman-gtk/usr/share/doc/HOWTO_network-setup.html

```diff
@@ -15,7 +15,7 @@
 
 	<div id="container">
 		<header class="logo-header">
-			<img class="header-logo" alt="Logo" src="puppylogo96.png">
+			<img class="header-logo" alt="Logo" src="/usr/share/doc/puppylogo96.png">
```

Comment:
Does this work without `file://` (`file:///usr/share/doc/puppylogo96.png`)? And why not simply `../puppylogo96.png`?

Reply:
> Does this work without `file://` (`file:///usr/share/doc/puppylogo96.png`)? 

Of course it does. HTML uses path similar to the OS, i.e. Linux here. Actually `file:///usr/share/doc/puppylogo96.png` won't work; it is just a URL used by browsers to know that the URL is actually a local file.

> And why not simply `../puppylogo96.png`?
Using relative links like this, you'll have to be extra careful about what should actually be there after the build has finished. Just for example- `../puppylogo96.png` is wrong and should be `./puppylogo96.png`. Basically, it uses more of your head 😁 (more thinking).

Reply:
@dimkr, I think - why change if it just works? I mean, holding the ears round the head and holding the ears directly is just the same?

Reply:
I don't think this really matters, it's a matter of style mostly.